services/users/src/routes/users.ts

```diff
@@ -1,7 +1,30 @@
 import express from 'express'
 import { userCollection } from '../collections/users'
+import { verifySession } from 'supertokens-node/recipe/session/framework/express'
+import { SessionRequest } from 'supertokens-node/framework/express'
 const router = express.Router()
 
+router.get('/', verifySession(), (req: SessionRequest, res: express.Response) => {
+    const address = req.session?.getUserId()
+    const user = userCollection.find(user => user.address === address?.toLowerCase())
+    res.setHeader('Content-Type', 'application/json')
+    res.status(200)
+    if (user) {
```

Comment:
Just return { user } or { error }.

Reply:
Not sure what you mean by this one. I simplified it I guess, but not sure it's what you want.